cmd/commands/deploy.go

```diff
@@ -189,10 +214,11 @@ func configureVersionInfo(ctx context.Context, a inngest.ActionVersion) (inngest
 
 	// If we're publishing a specific version, attempt to find it.  Else, load the latest
 	// action version.  This automatically happens depending on whether a.Version is nil.
-	found, err := state.Action(ctx, a.DSN, a.Version)
+	found, err := state.Client.Action(ctx, a.DSN, a.Version)
+
 	// When deploying without specifying an action, allow "action not found"
 	// errors.
-	if a.Version == nil && err != nil && err.Error() == "action not found" {
+	if a.Version == nil && err != nil && (err.Error() == "action not found" || err.Error() == coredata.ErrActionVersionNotFound.Error()) {
```

Comment:
Nit for later:  we can use `errors.Is(err, coredata.ErrActionVersionNotFound)` to check for this error (https://pkg.go.dev/errors).